src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/widgets/Counter.h"
+#include "systemtask/SystemTask.h"
+#include <components/motion/MotionController.h>
```

Comment:
use `<>` includes for includes outside of InfiniTime, for example system libraries or std-libraries. For InfiniTime header please use `""`

```suggestion
#include "components/motion/MotionController.h"
```

Reply:
thank you for the explanation & improvements on the code! :heart: fixed in 39235224f476390ad2e5ffc3418ca96885a491d1